backend/pyproject.toml

```diff
@@ -31,8 +31,8 @@ dependencies = [
     "python-socketio==5.9.0", # For log_events
     "social-auth-app-django==5.3.0", # For OAuth
     "social-auth-core==4.4.2", # For OAuth
-    "unstract-sdk~=0.32.0",
-    "unstract-adapters~=0.19.0",
+    "unstract-sdk~=0.33.0",
+    "unstract-adapters~=0.19.1",
```

Comment:
Isn't this 19.2 now @Deepak-Kesavan ?

Reply:
Thanks for the catch. Pushing the changes.